docs/features/DefaultInterfaceImplementation.md

```diff
@@ -134,7 +134,7 @@ II.10.4 Method implementat ion requirements
 II.12 Semantics of interfaces
 Interfaces can have static fields and methods, but they shall not have instance fields or 
 methods. Interfaces can define virtual methods, but only if those methods are abstract 
-(see Partition I and �II.15.4.2.4).
+(see Partition I and §II.15.4.2.4).
```

Comment:
I wasn't sure what caused this line to change. Did a line ending get changed?

Reply:
When I created the fork and edited the file inline in GitHub, it informed me that it had converted the file from Windows-1252 encoding to UTF-8. I presume that line is the artifact thereof, since I didn't touch it.

In the plaintext email sent to me of this convo, the § in the removed diff line is rendered as the replacement character U+FFFD �, so I reckon UTF-8 is correct.